citation.cff

```diff
@@ -0,0 +1,28 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
+title: "CHGNet: Pretrained universal neural network potential for charge-informed atomistic modeling"
+authors:
+  - family-names: Deng
+    given-names: Bowen
+  - family-names: Zhong
+    given-names: Peichen
+  - family-names: Jun
+    given-names: KyuJung
+  - family-names: Riebesell
+    given-names: Janosh
+  - family-names: Han
+    given-names: Kevin
+  - family-names: Bartel
+    given-names: Christopher J.
+  - family-names: Ceder
+    given-names: Gerbrand
+date-released: 2023-02-24
+repository-code: https://github.com/CederGroupHub/chgnet
+arxiv: https://arxiv.org/abs/2302.14231
+doi: 10.1016/j.commatsci.2012.10.028
```

Comment:
This DOI is not CHGNet?

Reply:
https://doi.org/10.1038/s42256-023-00716-3

Reply:
oops, copy-paste error. good thing you caught!